platformio.ini

```diff
@@ -24,40 +24,40 @@ platform = teensy
 board = teensy35
 framework = arduino
 upload_protocol = teensy-cli
-test_build_project_src = true
+test_build_src = true
 board_build.f_cpu = 24000000L
 build_flags = -D PRINT
 lib_ldf_mode = deep
 extra_scripts = extra_script.py
-test_port = /dev/ttyACM0
+; test_port = /dev/ttyACM0
```

Comment:
Please leave this uncommented. If you would like to modify an existing environment make a new one.